### PR TITLE
Bumping lang version 

### PR DESCRIPTION
**Pull request by manuranga**
## Purpose
Bumping lang version because automated one got closed

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,8 +1,7 @@
 org.gradle.caching=true
 puppycrawlCheckstyleVersion=8.18
 group=org.ballerinalang
-version=0.6.0-alpha6-SNAPSHOT
```
Comment (Kishanthan) [inline gradle.properties:4]: do we need to remove this?

Comment (daneshk) [inline gradle.properties:4]: version=0.6.0-alpha6-SNAPSHOT

Comment (manuranga) [reply gradle.properties:4]: added back. I saw `SNAPSHOT` and got confused.

**Comment by manuranga** [comment]
Closing because automated pr is the latest

**Pull request closed without merging**